osu.Game/Graphics/UserInterface/TernaryStateRadioMenuItem.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class TernaryStateRadioMenuItem : TernaryStateMenuItem
```

Comment:
Xmldoc this, something like "always toggles to true, even when true" is enough.

By itself is kind-of a useless class so I'm wondering if it could be made into a helper, or like a builder class which you can add a bunch of funcs to and it spits out the radio items. But I'm not sure how easy that's to do, so an xmldoc will suffice for now.

Reply:
Oops, missed that.

Aside, I originally had it as a parameter on the base class, but felt like explicitly making classes made the consuming code more legible. Agree that a builder may be what we want.